Gems/ROS2/Code/Source/Lidar/LidarSensorConfiguration.cpp

```diff
@@ -68,6 +86,55 @@ namespace ROS2
         }
     }
 
+    AZ::Crc32 LidarSensorConfiguration::SegmentationClassesChangeNotify()
+    {
+        bool unknown_present = false;
+        bool ground_present = false;
+        bool changed = false;
+        for (size_t i = 0; i < m_segmentationClasses.size(); i++)
```

Comment:
maybe create unorder set here to make code little more clear here.

Reply:
I'm not sure if this would improve much, I'm also handling user putting right class with wrong id.

Reply:
I changed this to range based loop